### DISCUSSION TITLE
Guide to all parameters

### DISCUSSION THREAD

**Discussion by Jason-Schneiderman** [MiguelRipoll23/homebridge-securitysystem#493] [Q&A]
Is there a guide to what all the parameters do? I can't find documentation for  siren_sensor_seconds and tripped_sensor_seconds along with many others.

**Comment by MiguelRipoll23** [comment]
Sorry for not explaining them, my bad.

It you set the value to 5 seconds for example, every five seconds the sensor will be triggered when the security system is tripped or triggered respectively.
